### PR TITLE
Move finding freetype to xlib's CMakeLists.txt

### PR DESCRIPTION
**Pull request by endoffile78**
This change is [Reviewable](https://reviewable.io/reviews/utox/utox/758)

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -171,12 +171,6 @@ elseif(APPLE)
     add_definitions("-x objective-c")
     add_subdirectory(src/cocoa)
 elseif(UNIX)
-    find_package(Freetype REQUIRED)
-    include_directories(${FREETYPE_INCLUDE_DIRS})
-    message("Found Freetype version ${FREETYPE_VERSION_STRING}")
-    message("Freetype include: ${FREETYPE_INCLUDE_DIRS}")
-    message("Freetype library: ${FREETYPE_LIBRARIES}")
```
Comment (cebe) [inline CMakeLists.txt]: afair this is here because it is used to link the utox binary below in this file: `${LIBRARIES}`

Comment (GrayHatter) [inline CMakeLists.txt]: I think it goes to the parent scope IIRC

Travis will tell us

Comment (cebe) [inline CMakeLists.txt]: it did not when I implemented this.

Comment (robinlinden) [inline CMakeLists.txt]: Looks like Travis likes it. OS X is failing because who knows. Nothing that this PR broke anyway.

**Comment by GrayHatter** [comment]
:lgtm:

---

Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/utox/utox/758#-:-KdOmfn4EiOa0n3b-9TG:bnfp4nl)*

**Comment by cebe** [comment]
:lgtm: verified, seems to work. Binary is linked correctly on debian.

---

Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/utox/utox/758#-:-KdPfAECFxVc4u84GWhP:b-7h3jl1)*

**Pull request merged**